dev/i18n/check_translations_completeness.py

```diff
@@ -467,9 +485,9 @@ def cli(language: str | None = None, add_missing: bool = False):
         summary,
     )
     if not found_difference and not has_todos:
-        console.print("[green]All translations are complete and consistent![/green]")
-    if found_difference:
-        sys.exit(1)
+        console.print("\n[green]All translations are complete and consistent![/green]\n\n")
+    else:
+        console.print("\n[red]Some translations are not consistent![/red]\n\n")
```

Comment:
```suggestion
        console.print("\n[red]Some translations are not complete nor consistent![/red]\n\n")
```
Could you please clarify what consistency is in this context? (it's not that clear)